apps/staking/locales/en.json

```diff
@@ -931,5 +948,143 @@
+    },
+    "switch": {
+      "label": "Vesting Mode",
+      "aria": "Toggle vesting mode"
+    },
+    "startDialog": {
+      "title": "Vesting Contract",
+      "description": "We noticed your wallet is connected to one or more vesting contracts. Select one to use otherwise skip this modal",
+      "descriptionSheet": "Select a vesting contract to use",
+      "continueButton": {
+        "label": "Continue",
+        "aria": "Use the selected vesting contract"
+      },
+      "skipButton": {
+        "label": "Skip",
+        "aria": "Skip vesting contract and continue using the connected wallet"
+      }
+    },
+    "infoDialog": {
+      "title": "Vesting Contract Info",
+      "address": "Contract Address",
+      "addressDescription": "Arbitrum Contract address for the vesting contract",
+      "initialBalance": "Initial Balance",
+      "initialBalanceDescription": "The initial amount of time-locked tokens in the contract",
+      "beneficiary": "Owner",
+      "beneficiaryDescription": "The wallet address that is the owner of the vesting contract",
+      "editBeneficiaryDisabledDescription": "The vesting contract does not allow changing the owner.",
+      "switchButton": {
+        "label": "Switch Contract",
+        "aria": "Switch to another vesting contract"
+      },
+      "disconnectButton": {
+        "label": "Disconnect",
+        "aria": "Disconnect from the vesting contract"
+      },
+      "endTimeEnded": "Unlocked"
+    },
+    "transferBeneficiaryDialog": {
+      "title": "Transfer Ownership",
+      "description": "Transfer the ownership of the vesting contract to another wallet.",
+      "disabledDescription": "The vesting contract does not allow transferring the owner.",
+      "currentBeneficiary": "Current Owner",
+      "currentBeneficiaryDescription": "The wallet address that is currently the owner of the vesting contract.",
+      "newBeneficiary": "New Owner",
+      "newBeneficiaryDescription": "The wallet address that will be the owner of the vesting contract.",
+      "errorCantBeSame": "The new owner cannot be the same as the current owner.",
+      "errorCantBeSelf": "The new owner cannot be the same as the contract address.",
+      "confirmHeading": "Are you sure?",
+      "confirmTextS1": "This will remove the current owner {ethAddress}",
+      "confirmTextS2": "and transfer ownership and control of this vesting contract to the new owner {ethAddress}",
+      "confirmTextS3": "<br></br>All funds will belong to the new address. This action is irreversible and cannot be undone.",
+      "transferButton": {
+        "label": "Transfer Owner",
+        "aria": "Transfer the owner of the vesting contract to another wallet."
+      },
+      "confirmButton": {
+        "label": "Confirm",
+        "aria": "Confirm transferring the owner of the vesting contract to another wallet."
+      },
+      "contractCall": {
+        "idle": "Transferring ownership of vesting contract to {newBeneficiary}",
```

Comment:
We seem to stop using the here. Probably best to stick to one style of writing 🤔

Reply:
this is in the status

Reply:
used "the"